lib/elixir/pages/Naming Conventions.md

```diff
@@ -0,0 +1,90 @@
+# Naming Conventions
+
+## Naming modules
+
+Elixir module names are written in "upper camel case", like `OptionParser`.
+
+Capital letters are kept in abbreviations, like `ExUnit.CaptureIO` or `Mix.SCM`.
```

Comment:
I tried it this morning in our app to confirm its still a problem and it was. We are on Elixir 1.1.1 FWIW. I haven't tried against Elixir 1.2 yet.